jenkins/bisect.jpl

```diff
@@ -390,16 +390,32 @@ def findMergeBase(kdir, good, bad) {
             script: "git merge-base --is-ancestor ${base} HEAD")
 
         if (good_base != 0) {
-            def ref = "${params.REF_KERNEL_TREE}/${params.REF_KERNEL_BRANCH}"
+
```

Comment:
Maybe do this here:
```javascript
def ref_url = params.REF_KERNEL_URL
def ref_tree = params.REF_KERNEL_TREE
def ref_branch = params.REF_KERNEL_BRANCH
```
then if any of them is not defined use `kci_build get_reference` as in the block below.

Reply:
Changed